### PR TITLE
ngfw-14476 updated feedback url

### PR DESCRIPTION
**Pull request by DhirajM09**
As part of the Untangle rebranding to Arista we need to update the feedback link in NGFW so that it points to edge.arista.com/feedback. Same has been changed in this PR.

Refer below screenshots regarding the change, tested locally.

Screenshot from 2024-01-30 12-53-42

On clicking Suggest Idea button in nav bar user will be redirected to edge.arista.com/feedback and eventually to feedback page

Screenshot from 2024-01-30 12-54-02

Screenshot from 2024-01-30 12-54-17

### CODE REVIEW

File: uvm/impl/com/untangle/uvm/UvmContextImpl.java [hunk 1]
```diff
@@ -1195,6 +1197,18 @@ public String getHelpUrl()
...
     }
 
+    /**
+     * getFeedbackUrl - get the feedback URL for the UI
+     * @return String
+     */
+    public String getFeedbackUrl()
+    {
+        String url = System.getProperty(PROPERTY_FEEDBACK_URL);
+        if (url == null)
```
Comment (mayur-dhande) [inline uvm/impl/com/untangle/uvm/UvmContextImpl.java:1207]: @cblaise-untangle don't we need do an empty/blank string test instead of just a null check ?

Comment (cblaise-untangle) [reply uvm/impl/com/untangle/uvm/UvmContextImpl.java:1207]: I believe System.gerProperty returns null if the value is not found.  But check to see if it does return a blank string and if so, yes, use that conditional.

File: uvm/servlets/admin/app/view/main/MainController.js [hunk 2]
```diff
@@ -120,7 +120,7 @@ Ext.define('Ung.view.main.MainController', {
     },
 
     suggestHandler: function (btn) {
-        var suggestUrl = Rpc.directData('rpc.helpUrl') + '?fragment=feedback&' + Util.getAbout();
+        var suggestUrl = Rpc.directData('rpc.feedbackUrl');
         window.open(suggestUrl);
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/app/view/main/MainController.js:124]: Add the same parameters to the new URL.

**Review by cblaise-untangle — CHANGES_REQUESTED** [review]
Add a new test to test_uvm.py to verify the new url.  It will be a pretty simple test.

You can verify this test via VS by running the test operation. You only need to run the single test (not the whole suite) and can do so with a command like "-t uvm  -T <name of new test>"

File: uvm/impl/com/untangle/uvm/UvmContextImpl.java (hunk 1 above)
Comment (DhirajM09) [reply uvm/impl/com/untangle/uvm/UvmContextImpl.java:1207]: @mayur-dhande @cblaise-untangle I confirmed that System.getProperty returns null if value not found. only null check should be fine.

File: uvm/servlets/admin/app/view/main/MainController.js (hunk 2 above)
Comment (DhirajM09) [reply uvm/servlets/admin/app/view/main/MainController.js:124]: With parameters it is navigating to page shown below, with message : **Nothing found for this page. Try a search instead?**

I think those parameters were required for wiki link to navigate to correct page as  same wiki link was used for help and feedback.

Screenshot from 2024-01-31 11-45-04

Comment (cblaise-untangle) [reply uvm/servlets/admin/app/view/main/MainController.js:124]: Got it.  I see the problem.  Ok, this PR is approved.

**Review by cblaise-untangle — APPROVED** [review]

**Review by cblaise-untangle — APPROVED** [review]
